guide/10-mapping-and-visualization/working-with-web-maps-and-web-scenes.ipynb

```diff
@@ -1 +1,816 @@
iv></p>\",\n", "       'title': \"<div><p style='font-weight:bold;'><span>{typhoonclass} {typhoon}</span></p></div>\",\n", "       'type': 'image',\n", "       'value': {'sourceURL': '{image}'}}],\n", "     'title': '{typhoon}'},\n", "    'title': 'Typhoons Q2',\n", "    'url': 'http://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/PacificTyphoons2005_WFL/FeatureServer/4',\n", "    'visibility': False}],\n", "  'listMode': 'hide-children',\n", "  'opacity': 1,\n", "  'title': 'April - June',\n", "  'visibility': True,\n", "  'visibilityMode': 'inherited'}]"]}, "execution_count": null, "metadata": {}, "output_type": "execute_result"}], "source": ["webscene_obj['operationalLayers']"]}, {"cell_type": "markdown", "metadata": {}, "source": ["Similar to a `WebMap` seen earlier, you can find the list of all keys that you can query for by calling the keys() function:"]}, {"cell_type": "code", "execution_count": null, "metadata": {}, "outputs": [{"data": {"text/plain": ["odict_keys(['authoringApp', 'baseMap', 'initialState', 'version', 'operationalLayers', 'presentation', 'authoringAppVersion'])"]}, "execution_count": null, "metadata": {}, "output_type": "execute_result"}], "source": ["webscene_obj.keys()"]}, {"cell_type": "markdown", "metadata": {}, "source": ["### Displaying the web scene\n", "When using the Jupyter notebook environment, a `WebScene` object can be easily visualized by simply querying it. A map widget loads up and the scene is rendered:"]}, {"cell_type": "code", "execution_count": null, "metadata": {"collapsed": true}, "outputs": [], "source": ["webscene_obj"]}, {"cell_type": "markdown", "metadata": {}, "source": ["![tropical cyclones web scene](http://esri.github.io/arcgis-python-api/notebooks/nbimages/05_Using_updating_GIS_06.PNG)"]}, {"cell_type": "markdown", "metadata": {}, "source": ["### Updating a web scene\n", "The `WebScene` object provides an `update()` method that allows you to modify a web scene and save those changes back. To update a web scene, you modify its definition by changing the adding, removing or changing the values of its keys and then call the update() method. Refer to the sample notebook on [using and updating a web scene](https://developers.arcgis.com/python/sample-notebooks/using-and-updating-gis-content/#Using-and-updating-a-web-scene) for an example."]}], "metadata": {"anaconda-cloud": {}, "kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.8.2"}, "toc": {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": true, "toc_position": {}, "toc_section_display": true, "toc_window_display": true}}, "nbformat": 4, "nbformat_minor": 1}
```

Comment:
<p>The 7th web scene from this list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1896/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>The <code>Recent Storm Scene</code> item is in index position <code>8</code>. It's actually the 9th item in the list, but since lists are 0 indexed, the item is retrieved with <code>webscene_search[8]</code>.   </p>